src/components/approval-tasks/ApprovalRow.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { Split, SplitItem, Tooltip } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector } from 'react-redux';
+import {
+  getGroupVersionKindForModel,
+  RowProps,
+  ResourceLink,
+  TableData,
+  Timestamp,
+  UserInfo,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { tableColumnClasses } from './approval-table';
+import './ApprovalRow.scss';
+import { ApprovalTaskKind } from '../../types';
+import {
+  ApprovalTaskModel,
+  NamespaceModel,
+  PipelineRunModel,
+} from '../../models';
+import {
+  getApprovalStatus,
+  getApprovalStatusInfo,
+  getPipelineRunOfApprovalTask,
+} from '../utils/pipeline-approval-utils';
+import { ApprovalStatusIcon } from '../pipeline-topology/StatusIcons';
+import { PipelineRunKind } from '../../types';
+import ApprovalTasksKebab from './ApprovalTasksKebab';
+import { SDKStoreState } from '@openshift-console/dynamic-plugin-sdk/lib/app/redux-types';
+
+const ApprovalRow: React.FC<
+  RowProps<
+    ApprovalTaskKind,
+    {
+      pipelineRuns: PipelineRunKind[];
+    }
+  >
+> = ({ obj, activeColumnIDs, rowData: { pipelineRuns } }) => {
+  const { t } = useTranslation('plugin__pipelines-console-plugin');
+  const {
+    metadata: { name, namespace, creationTimestamp },
+    spec: { description, numberOfApprovalsRequired },
+    status: { approvers, approversResponse },
+  } = obj;
+
+  const [user, setUser] = React.useState<UserInfo>(
+    useSelector((state: SDKStoreState) => state.sdkCore.user),
+  );
+  if (user.username === 'kube:admin') {
+    setUser({ ...user, username: 'kubernetes-admin' });
+  }
```

Comment:
You must not change the user (in redux) here!

If you want switch the username from `kube:admin` to `kubenetes-admin` (I don't see a reason do to this...) please do something like this:

```ts
let username = user.username;
if (username === 'kube:admin') {
  username = 'kubernetes-admin';
}
```

But, why is there a need for this at all?